controller/Dockerfile

```diff
@@ -3,20 +3,20 @@ ARG DEBIAN_VERSION="11.7"
 FROM --platform=$BUILDPLATFORM ubuntu:24.04 as builder
 
 RUN mkdir -p \
-    /artifacts/linux/amd64 \
-    /artifacts/linux/arm64
+    /artifacts/localbin
 
-COPY linux/amd64/kube-guardian /artifacts/linux/amd64
-COPY linux/arm64/kube-guardian /artifacts/linux/arm64
+COPY localbin/kube-guardian /artifacts/localbin
```

Comment:
Lets use the --chown and --chmod etc on the COPY command to ensure we have correct perms